Agile Essay Re-reference.tex

```diff
@@ -0,0 +1,68 @@
+
+\section{Trello board}
+"The Development Team usually starts by designing the system and the work needed to convert the Product Backlog into a working product Increment. Work may be of varying size, or estimated effort. However, enough work is planned during Sprint Planning for the Development Team to forecast what it believes it can do in the upcoming Sprint." \cite{scrum alliance}
+
+Before the sprint the team will contribute towards a board of tasks which will be designed to have multiple stages. Throughout these stages each task can vary in size and also some tasks will be prioritised over others depending on how intense each task is. By eliminating these tasks first, although the work load will be the same, it will allow the team to evaluate the task in review which may then be taken back a stage for another member to adapt to complete to the highest standard. By doing so it enables the reprioritised task which may be more valuable to the project won’t be rushed and can be completed to a near perfect standard while also allowing other tasks to be completed on time.
+
+\section{Development skills}
+"Scrum emphasizes empirical feedback, team self-management, and striving to build properly tested product increments within short iterations." \cite{agile methodology rss}
+
+Scrum can provide the team with multiple skills that will assist them later on with future projects and endeavours. The team will learn to review completed tasks that have been achieved by their peers, by doing so they can give a critical analysis of the work completed and improve upon it where needed. The team will also self-manage themselves when the scrum is in practise. Throughout the team will complete various pieces of work individually, at their own pace but should be keeping up to date with the sprint deadline set by the scrum master. The team will also push themselves to complete work as efficient as possible but up to the highest standard of which they can achieve to work within the brief, rather than assigning each team member a specific role they will complete work by one piece at a time and then move onto the next piece of work. By doing so, this spreads out the workload of the sprint amongst the team and also therefore reduces the stress levels amongst the team.
+
+\section{Cons}
+Although there are many benefits to scrum, there are also many cons to it too. 
+"The Cons
+•	Scrum often leads to scope creep, due to the lack of a definite end-date
+•	The chances of project failure are high if individuals aren't very committed or cooperative
+•	Adopting the Scrum framework in large teams is challenging
+•	The framework can be successful only with experienced team members
+•	Daily meetings sometimes frustrate team members
+•	If any team member leaves the project in the middle, it has a huge negative impact on the project
+•	Quality is hard to implement, until the team goes through aggressive testing process" \cite{2017}

```

Comment:
Use the `itemize` environment for bullet lists. See the [list guide](https://www.sharelatex.com/learn/Lists).